api/v1alpha1/backendtrafficpolicy_types.go

```diff
@@ -86,6 +86,16 @@ type BackendTrafficPolicySpec struct {
 	//
 	// +optional
 	Timeout *Timeout `json:"timeout,omitempty"`
+
+	// The compression config for the backend connections.
```

Comment:
compression/decompression will reduce the body size, at the same it will increase the CPU usage, right?

Reply:
yes, it is all about the tradeoff I think.

Reply:
can you point out this in the api doc?

Reply:
yea, will do. thanks for the review!